istio/istio.spec

```diff
@@ -14,7 +14,7 @@
 %global debug_package   %{nil}
 %endif
 
-%global git_commit c0b8cb379eacaf876402da54c86e278e8b05cb28
+%global git_commit c5db59a9076a5b7bf968ae8f69cc786b3e015d78
```

Comment:
Should this be 7f80b7e965bb4b5ab0f777b0ac6f6049adb0dab4?

Reply:
The commit matches the product side which was for the last code change, there has been one further commit in the istio repo which was only for the charts so would not impact the RPMs.